pkg/route/api/v1/defaults.go

```diff
@@ -2,6 +2,18 @@ package v1
 
 import "k8s.io/kubernetes/pkg/runtime"
 
+func SetDefaults_RouteSpec(obj *RouteSpec) {
+	if len(obj.WildcardPolicy) == 0 {
+		obj.WildcardPolicy = "None"
+	}
+	switch obj.WildcardPolicy {
+	case WildcardPolicyType("None"):
+		obj.WildcardPolicy = WildcardPolicyNone
+	case WildcardPolicyType("Subdomain"):
+		obj.WildcardPolicy = WildcardPolicySubdomain
+	}
+}
```

Comment:
I don't know too much about this part - AFAICT, it was just setting defaults if no values were set on fields in an object. And there's no errors thrown back/logging here in similar defaults code.   @liggitt  may be able to tell more.